### DISCUSSION TITLE
How to disable STRIMZI_NETWORK_POLICY_GENERATION

### DISCUSSION THREAD

**Discussion by jackchuong** [strimzi/strimzi-kafka-operator#9126] [Q&A] [answered]
Hi all,
I installed strimzi kafka operator by 
`kubectl apply -f kafka-cdrs.yaml # kafka-cdrs.yaml downloaded from https://strimzi.io`
I want to to disable network policy generation so it won't create network policy resouces, how can I do that ?
I have to edit deployment.apps/strimzi-cluster-operator and set env STRIMZI_NETWORK_POLICY_GENERATION = false manually ? Is there any way else ?

**Comment by scholzj** [comment, marked as answer]
Yes, you can edit the deployment.
